src/packet_handlers/peer_subscription.rs

```diff
@@ -16,6 +16,15 @@ pub fn handle_peer_packet<M: Messenger, P: PlayerState>(
                 messenger.broadcast_packet(Packet::SpawnPlayer(packet), None, false);
             }
         }
+        Packet::DestroyEntities(packet) => {
+            assert!(
+                packet.entity_ids.len() == 1,
+                "Cannot handle entity destroy packets from peers with multiple ids"
+            );
+            if packet.entity_ids[0] >= 1000 {
+                messenger.broadcast_packet(Packet::DestroyEntities(packet), None, false);
+            }
+        }
```

Comment:
We want to ignore messages that say that our anchored players have been deleted. We keep track of those players ourselves, and they have in fact not been deleted from our perspective, they've just walked to a new map. There are already some comments in here about it, but sourcing the events coming from peers is an area that needs work. Here's the ticket for that: https://github.com/DuncanUszkay1/Patchwork/issues/130